cppcache/integration/test/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ add_executable(cpp-integration-test
   ExampleTest.cpp
   ExpirationTest.cpp
   FunctionExecutionTest.cpp
+  PartitionRegionOpsWithRedundancyAndServerGoesDown.cpp
```

Comment:
I think it general we've been having test files end in `Test` just to make it super obvious when browsing files it is a test.  Also the file reads more like a test case rather than an identifier for a set of tests.  I think the test file being called `PartitionRegionOpsTest.cpp` would work since you use `getPartitionedRegionWithRedundancyServerGoesDown` and putPartitionedRegionWithRedundancyServerGoesDown` as your test names.  We can see if others have thoughts about naming

Reply:
I agree with you.

Reply:
I tend to always work in an IDE that supports project view (Visual Studio, Xcode), wherein I already know I'm working with a test project. Hence the Test suffix is a bit redundant. Also, we already have 4 tests in the new framework that don't use the Test suffix.